docs/v0.9/en/user-guide/operations/disaster-recovery/dr-solution-based-on-dual-active-standby.md

```diff
@@ -1 +1,48 @@
-# DR solution based on Dual Active-Standby
\ No newline at end of file
+# DR solution based on Dual Active-Standby
+
+## RPO
+
+In GreptimeDB's "Dual Active-Standby" architecture, there are two nodes. Both nodes can provide the ability to execute reads and writes from clients. However, to achieve the goals of the RTO and RPO, we need to do some configurations about them. First of all, let me introduce the modes to reads and writes.
+
+For reads:
+
+- `SingleRead`: a read operation is executed in one node only, and the results are returned to the client directly.
+- `DualRead`: a read operation is executed in both nodes. The results are merged and deduplicated before returning to the client.
+
+The following picture illustrates the difference between the two read modes:
+
+![disaster-recovery-read-mode](/disaster-recovery-read-mode.png)
+
+For writes:
+
+- `SyncWrite`: a write operation is executed in both nodes, and is considered success only if it is succeeded in both nodes before returning to the clients.
+- `AsyncWrite`: a write operation is still executed in both nodes, but the result is returned to the clients when it is done on the active node. The standby node will receive a copy of the write operation from the active node, asynchronously.
+
+The following picture illustrates the difference between the two write modes:
+
+![disaster-recovery-write-mode](/disaster-recovery-write-mode.png)
+
+So there are four combinations on reads and writes, and their RPOs are:
+
+| RPO | `SingleRead` | `DualRead` |
+| - | - | - |
+| `SyncWrite` | 0 | 0 |
+| `AsyncWrite` | network latency between two nodes | 0 |
+
+Since the writes are synchronized between the two nodes in `SyncWrite` mode, the RPO is always zero regardless what read mode is. However, `SyncWrite` requires both nodes functioned well simultaneously to handle writes. If your workload is write-light-read-heavy, and can tolerate some system unavailable time to bring back the healthiness of both nodes should the disaster happened, we recommend the `SyncWrite + SingleRead` combination.
```

Comment:
```suggestion
Since the writes are synchronized between the two nodes in `SyncWrite` mode, the RPO is always zero regardless what read mode is. However, `SyncWrite` requires both nodes functioned well simultaneously to handle writes. If your workload is write-light-read-heavy, and can tolerate some system unavailable time to bring back the healthiness of both nodes when the disaster happened, we recommend the `SyncWrite + SingleRead` combination.
```